### DISCUSSION TITLE
Using dotenv in a playwright test file

### DISCUSSION THREAD

**Discussion by vikramjb** [artilleryio/artillery#2515] [Artillery]
I have many scenarios that I am using to test my app on. And I store all these env variables on env file. I am using --dotenv command line parameter but I can't see to find an example where I can use the variables in env file getting used in the playwright test script. Is there a sample somewhere that I can take a look?

**Comment by vikramjb** [comment]
I found the answer for it. They were available on context.vars. Thank you 👍

**Comment by bernardobridge** [comment]
For reference, they're also available under `$env` to the test script, as they'll just be regular environment variables in the environment. So at any point in the YAML test script (scenarios or config sections), you can use `{{ $env.VAR_NAME }}`. And in Javascript custom code, you can use `process.env.VAR_NAME` as usual.

Reference: https://www.artillery.io/docs/reference/test-script#using-environment-varables
